test/e2e/traffic_manager_test.go

```diff
@@ -180,6 +194,378 @@ var _ = Describe("Test exporting service via Azure traffic manager", func() {
 			atmValidator.ValidateProfile(ctx, atmProfileName, atmProfile)
 		})
 	})
+
+	Context("Test invalid trafficManagerBackend (invalid profile)", Ordered, func() {
+		var backend fleetnetv1alpha1.TrafficManagerBackend
+		var backendName types.NamespacedName
+		memberDNSLabels := make([]string, 2)
+
+		BeforeEach(func() {
```

Comment:
Should we use BeforeAll here?

Reply:
in the describe(), i'm using beforeEach and afterEach, these statements will be run for each IT, so here it should use beforeEach and afterEach to reconfig service.